llvm/lib/Transforms/Scalar/ConstraintElimination.cpp

```diff
@@ -1067,6 +1067,8 @@ void State::addInfoFor(BasicBlock &BB) {
     case Intrinsic::smin:
     case Intrinsic::smax:
       WorkList.push_back(FactOrCheck::getInstFact(DT.getNode(&BB), &I));
+      WorkList.push_back(
```

Comment:
comment above needs updating now, as this queues for simplification now. Also, `abs` shouldn't be queued for simplification, right?

Reply:
Done.